libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/LanguageUtils.java

```diff
@@ -1,24 +1,43 @@
 package org.wordpress.android.util;
 
 import android.content.Context;
+import android.support.annotation.Nullable;
 
 import java.util.Locale;
 
 /**
  * Methods for dealing with i18n messages
  */
 public class LanguageUtils {
```

Comment:
I was trying to understand the deprecated methods in this class, but I feel like we should drop this all together if we are not going to use it. What do you think?

P.S: I don't think it has to happen in this PR, I'd actually prefer a separate one for this.